### ISSUE TITLE
The utftexindy.lua does not work for languages 'slovak-large' and 'slovak-small'.

### ISSUE THREAD

**Issue by gszasz** [michal-h21/iec2utf#1]
The language 'slovak' exists in the latest stable _xindy_ release (2.3) in two different definitions:
- slovak-large
- slovak-small

The 'slovak-large' definition is pretty close to the official alphabetical ordering in Slovak language (although still not fully correct).  The problem is that language module for the 'slovak-large' language is named `lang/slovak/large-utf8-lang` that does not fit the expectations of the `utftexindy.lua` script.

**Comment by michal-h21** [comment]
Thanks for the report, I've added support for this feature, just try to use `utftexindy -L slovak/large`.

**Comment by gszasz** [comment]
That's indeed very elegant and simple solution.  Nevertheless, it doesn't follow _xindy_'s language-name conventions (note, that the official syntax is `texindy -L slovak-large`) and thus usage of `utftexindy.lua` script may lead into an unnecessary confusion.  I'd tweak your implementation a little bit:

``` lua
-- support languages with multiple alphabet rule variants
-- ie. slovak-large, slovak-small, spanish-modern, spanish-traditional,... etc.
-- see http://xindy.sourceforge.net/doc/make-rules-alphabets-doc.pdf
local langsep = "/"

if lang:match "-" then langsep = "-" end

local langmodule = "lang/"..lang:gsub("-", "/")..langsep.."utf8-lang"
```

(See #2 for more details)

**Comment by gszasz** [comment]
The fix #2 works for me. I think we can close this issue.
